Visualizer-Web-App/src/app/app.component.html

```diff
@@ -18,18 +18,20 @@
                 <div *ngIf="toggle" class="col s3 m3 l3"><youtube-dl></youtube-dl></div>
                 <div *ngIf="!toggle" class="col s3 m3 l3"><file-upload></file-upload></div>
             </td>
-            <td class="col s3 m3 l3 greyed-out-fingerprint fixed-action-btn click-to-toggle">
-                <a class="btn-floating btn-large waves-effect waves-blue waves-light blue" >
-                    <i class="material-icons">fingerprint</i>
-                </a>
-                <ul>
-                    <li><a class="btn-floating btn-large waves-effect waves-red red pulse" onclick="Materialize.toast('Really disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_very_dissatisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-orange orange pulse" onclick="Materialize.toast('Disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_dissatisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-yellow yellow darken-1 pulse" onclick="Materialize.toast('Liked...', 3000, 'rounded')"><i class="material-icons">sentiment_satisfied</i></a></li>
-                    <li><a class="btn-floating btn-large waves-effect waves-green green pulse" onclick="Materialize.toast('Loved...', 3000, 'rounded')"><i class="material-icons">sentiment_very_satisfied</i></a></li>
-                    <li><slide-menu></slide-menu></li>
-                </ul>
-            </td>
+
         </tr>
     </table>
 </div>
+
+<div class="greyed-out-fingerprint fixed-action-btn click-to-toggle">
+    <a class="btn-floating btn-large waves-effect waves-blue waves-light blue" >
+        <i class="material-icons">fingerprint</i>
+    </a>
+    <ul>
+        <li><a class="btn-floating btn-large waves-effect waves-red red pulse" onclick="Materialize.toast('Really disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_very_dissatisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-orange orange pulse" onclick="Materialize.toast('Disliked...', 3000, 'rounded')"><i class="material-icons">sentiment_dissatisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-yellow yellow darken-1 pulse" onclick="Materialize.toast('Liked...', 3000, 'rounded')"><i class="material-icons">sentiment_satisfied</i></a></li>
+        <li><a class="btn-floating btn-large waves-effect waves-green green pulse" onclick="Materialize.toast('Loved...', 3000, 'rounded')"><i class="material-icons">sentiment_very_satisfied</i></a></li>
+        <li><slide-menu></slide-menu></li>
+    </ul>
+</div>
```

Comment:
I wasn't having an issue like that when I committed those changes. The conflicts were just that we had worked on the same files, I manually resolved them. But I am having some issues with weird slide menu behavior, after trying to change it a bit. I'll keep working on it.
Also, I think we should leave this pull request open until this branch is closer to finished or we'll end up just making lots pushes to master.